cmds/identityd/main.go

```diff
@@ -454,18 +454,25 @@ func bcdbClient() (client.Directory, error) {
 
 func registerNode(nodeID pkg.Identifier, farmID pkg.FarmID, version string, store client.Directory, loc geoip.Location) error {
 	log.Info().Str("version", version).Msg("start registration of the node")
+	var err error
 
 	v1ID, _ := network.NodeIDv1()
 
 	publicKeyHex := hex.EncodeToString(base58.Decode(nodeID.Identity()))
 
+	hostName, err := os.Hostname()
+	if err != nil {
+		return err
```

Comment:
Maybe not return here, instead set hotname to `<unknown>` or something. It's better than not having the node registered on the explorer

Reply:
good catch